elements/lisk-transactions/src/unlock_token.ts

```diff
@@ -85,7 +88,9 @@ export const unlockToken = (
 		return baseTransactionToJSON(transaction as UnlockTransaction);
 	}
 
-	const unlockTransaction = new UnlockTransaction(transaction as UnlockTransaction);
+	const unlockTransaction = new UnlockTransaction(
+		transaction as UnlockTransaction,
```

Comment:
I'm getting TypeError `Conversion of type '{ type: number; asset: { unlockObjects: readonly Unlock[]; }; nonce: bigint; fee: bigint; senderPublicKey: string | undefined; }' to type 'UnlockTransaction' may be a mistake because neither type sufficiently overlaps with the other`

Reply:
it works now